.github/actions/build_package/action.yml

```diff
@@ -67,9 +66,9 @@ runs:
         Section: base
         Priority: optional
         Architecture: $arch
-        Maintainer: Polygon Devops <devops@polygon.technology>
+        Maintainer: Miden <sergerad@miden.team>
```

Comment:
I wonder if we should create a catch-all team email, `node@miden.team` or `devops@miden.team`.

This would also be useful for security reports.